### DISCUSSION TITLE
Should we build a leaderboard?

### DISCUSSION THREAD

**Discussion by atapas** [reactplay/react-play#142] [Ideas]
Hey All,

What do you think? Should we build a leaderboard? If yes, based on what?

Some ideas are,

A) Number of plays
B) Number of plays of different scores based on levels
C) Numer of plays of different scores based on levels + number of likes
D) Numer of plays of different scores based on levels + number of likes + number of comments(engagements)
E) Anything else?

Based on the algorithm we come up with, we can provide points to the users. That point decides the leaderboard.

**Comment by atapas** [comment]
@hasnainmakada-99 Do you have further ideas on this implementation?

**Reply by hasnainmakada-99** [reply]
No tapas sir the points you just described are good to go

**Reply by hasnainmakada-99** [reply]
If I have any further ideas, I'll reach out to you

**Comment by shrilakshmishastry** [comment]
Hello @atapas,
Yes, building a leaderboard would give more encouragement to build plays. I guess option B would go well and is there a measure to evaluate code quality?

**Reply by atapas** [reply]
I’m also thinking to get started with Option B. It is simpler to track too.

we need to have a qualification criteria for levels. Right now it is added randomly for the existing plays.

**Reply by vasantisuthar** [reply]
I suppose the likes and engagements of the play should also be a criterion for analyzing the leaderboard so that we can also get to know how much the play is helpful and interesting to other users. What do you think?

**Reply by hasnainmakada-99** [reply]
Yup that sounds a good Idea @vasantisuthar

**Reply by atapas** [reply]
Sounds good.

**Comment by MOHAMED-EHAB-DEV** [comment]
Hi @atapas 

I have a small idea:
1. We can give a certificate to the person who done a challenge
